backend/src/api/user/user.api.ts

```diff
@@ -1,17 +1,58 @@
 import { Router } from 'express';
 import { ApiPath, HttpCode, UsersApiPath } from '~/common/enums';
+import * as userService from '../../services/api/userService'
```

Comment:
```suggestion
import * as userService from '../../services/api/userService'
```
When you will add the `class` it should be something like 
`import { userService } from '~/services'`